### DISCUSSION TITLE
Start Backup Integrity Check from an external script?

### DISCUSSION THREAD

**Discussion by gadric** [nextcloud/all-in-one#1076] [Questions] [answered]
It is possible, comparable to daily backups, to trigger a backup integrity check from an external script?

**Comment by szaimen** [comment]
You mean this option?

<details>
<summary>click here</summary>

![image](https://user-images.githubusercontent.com/42591237/187035082-2b4d83dd-dab3-4a49-84f9-4133ba6bc2bd.png)

</details>

**Reply by gadric** [reply]
Yes exactly. Since this check takes a lot of time. I would like to schedule it at some time in the night :)

**Reply by szaimen** [reply]
So you use AIO's built-in backup solution and currently trigger daily backups from an external script using https://github.com/nextcloud/all-in-one#how-to-stopstartupdate-containers-or-trigger-the-daily-backup-from-a-script-externally?

**Reply by gadric** [reply]
Yes. The external scripts works as a charme. I want to regularly check the integrity. I started the check, but it takes over 2 hours. Would be nice if this can be scheduled via a script.

**Comment by szaimen** [comment, marked as answer]
Will be handled in https://github.com/nextcloud/all-in-one/issues/1077

**Reply by gadric** [reply]
Thanks !!!!!!

**Reply by szaimen** [reply]
This is now released with v2.0.2 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by gadric** [reply]
Looks good. Started from the cmd and is running as expected. If something goes wrong, I will report. Otherwise consider it successful. 

quick question: I switch to Beta in the Mastercontainer. I accidentally upgrade also the other containers. Now they have been switched to beta channel. Can I "downupgrade" to stable for all containers by switching the mastercontainer to stable again?

**Reply by Zoey2936** [reply]
Just wait until it is pushed to latest, then you can switch back
